backend/src/appointment/routes/schedule.py

```diff
@@ -112,6 +112,7 @@ def create_calendar_schedule(
     db_schedule = repo.schedule.create(db=db, schedule=schedule)
 
     # If slug isn't provided, give them the last 8 characters from a uuid4
+    # TODO: We might want to set initial slug to None if the user should start with an username only url
     slug = repo.schedule.generate_slug(db, db_schedule.id)
```

Comment:
I didn't know the flow here, currently there is a slug generated when first creating a schedule.

Reply:
This is now changed to only generate slugs if we have more than 1 schedule and no slug was initially given.